tests/Unit/Application/Regulation/Query/GetRegulationOrdersToDatexFormatQueryHandlerTest.php

```diff
@@ -98,7 +98,7 @@ public function testGetAll(): void
             ->willReturn('56456ff6-7e1c-4d24-aa09-9c650d7f6115');
         $regulationOrder1
             ->expects(self::once())
-            ->method('getDescription')
+            ->method('getTitle')
             ->willReturn('Description 1');
```

Comment:
(Détail)

```suggestion
            ->willReturn('Titre 1');
```

etc sur les autres "Description N" de ce fichier